pages/configuration/theme.mdx

```diff
@@ -1,9 +1,41 @@
-### Theme
+# Theme
 
 The default theme for users can be set via `config/options.json`. This feature sets the initial theme for users as **light**, **dark**, or **system**. The _Toggle Theme_ button can also be enabled or disabled here.
 
 **Note:** Theme settings are stored in the users local browser storage. Setting the theme will only affect new users to your site. It will not change the default theme for users who have already visited.
 
+## Properties
+
+| Property        | Type                      | Description                                                  |
+| --------------- | ------------------------- | ------------------------------------------------------------ |
+| `defaultTheme`  | `light`, `dark`, `system` | Sets the default theme for users.                            |
```

Comment:
Are `light`, `dark`, `system` types or options?  Shouldn't type just be `string`?  Would making an additional column like `choices` or `options` make sense?

Reply:
These are technically enums so I want to be clear about that. See https://www.radix-ui.com/primitives/docs/components/scroll-area#scrollbar where `orientation` is clearly defined to have specific accepted values.

What if I make a tooltip popover here?

`enum`  ( i ) --> `"light" | "dark" | "system"`

Reply:
@markpbaggett  Cool, that's baked into this PR now if you want to review that final part. I created a new Tooltip component for reuse.